examples/django_chat/chat/management/commands/runserver_socketio.py

```diff
@@ -0,0 +1,108 @@
+from re import match
+from _thread import start_new_thread
+from time import sleep
+from os import getpid, kill, environ
+from signal import SIGINT
+import six
+import copy
+
+from django.conf import settings
+from django.core.handlers.wsgi import WSGIHandler
+from django.core.management.base import BaseCommand, CommandError
+from django.core.management.commands.runserver import naiveip_re, DEFAULT_PORT
```

Comment:
I believe the only issue when running your example under Django 1.9 is that `DEFAULT_PORT` isn't defined. At least that's what I see here when I test.